lib/redbird/plug/session/redis.ex

```diff
@@ -14,26 +14,43 @@ defmodule Plug.Session.REDIS do
     opts
   end
 
-  def get(_conn, redis_key, _init_options) do
-    case get(redis_key) do
+  def get(_conn, namespaced_key, _init_options) do
+    case get(namespaced_key) do
       :undefined -> {nil, %{}}
-      value -> {redis_key, value |> :erlang.binary_to_term}
+      value -> {namespaced_key, value |> :erlang.binary_to_term}
     end
   end
 
   def put(conn, nil, data, init_options) do
     put(conn, generate_random_key(), data, init_options)
   end
   def put(_conn, redis_key, data, init_options) do
-    setex(%{key: redis_key, value: data, seconds: session_expiration(init_options)})
-    redis_key
+    key = add_namespace(redis_key)
```

Comment:
Minor: what do you think about inlining this function call instead of assigning the variable out?